dev-libs/jemalloc/jemalloc-5.3.0_pre20220405.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
8 should be available.

Reply:
hmm, yes, this is not being depended on by base-system packages so 8 should be fine

Reply:
This is indeed a good principle to have, but with EAPI-8 that boat has sailed already :) We have stable EAPI-8 base-system packages with no EAPI-7 ebuilds left in the tree.

let's all do better for the next EAPIs.